website/content/en/docs/olm-integration/generation.md

```diff
@@ -123,6 +123,29 @@ package manifest YAML file containing channel-to-version mappings, much like a b
 If your Operator is already formatted as a package manifests and you do not wish to migrate to the bundle format yet,
 you should add the following to your `Makefile` to make development easier:
 
+**For Go-based Operator projects**
+
+```make
+# Options for "packagemanifests".
+ifneq ($(origin FROM_VERSION), undefined)
+PKG_FROM_VERSION := --from-version=$(FROM_VERSION)
+endif
+ifneq ($(origin CHANNEL), undefined)
+PKG_CHANNELS := --channel=$(CHANNEL)
+endif
+ifeq ($(IS_CHANNEL_DEFAULT), 1)
+PKG_IS_DEFAULT_CHANNEL := --default-channel
+endif
+PKG_MAN_OPTS ?= $(FROM_VERSION) $(PKG_CHANNELS) $(PKG_IS_DEFAULT_CHANNEL)
+
+# Generate package manifests.
+packagemanifests: kustomize manifests
+  operator-sdk generate kustomize manifests -q
+  $(KUSTOMIZE) build config/manifests | operator-sdk generate packagemanifests -q --version $(VERSION) $(PKG_MAN_OPTS)
+```
```

Comment:
Ouch - kind of a bummer that we have to duplicate this entire block because of the single `manifests` dependency on the `packagemanifests` target.

This is one of those things that makes organization on our docs site extremely difficult when dealing with three operator types.

I'm almost to the point where I could be convinced to duplicate this (or some portion of this) doc (and other docs with similar issues) for each operator type.

This is okay for now, but definitely something we could think about improving later.

Reply:
I agree that we need to clean up the docs .. keeping it mind for follow-ups.